lib/linter-eslint.js

```diff
@@ -138,6 +143,11 @@ export default {
       try {
         results = JSON.parse(output).results
         messages = results[0].messages || []
+        // clobber ignored message based on settings
+        if (messages.length === 1 && !!~messages[0].message.indexOf('ignored because of your .eslintignore file') &&
```

Comment:
Yes that's cleaner, although the whole idea of linting unnecessarily should be avoided.
I'll update PR...